modules/veteran_verification/app/controllers/veteran_verification/v0/keys_controller.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+require 'base64'
+require_dependency 'notary'
+
+module VeteranVerification
+  module V0
+    # KeysController returns a set of public keys as Base64URL encoded PEMs
+    # that can be matched based on the kid (key id) and used to verify
+    # payloads signed by the same Notary in other endpoints.
```

Comment:
```suggestion
    # KeysController returns a JSON Web Key Set (JWKS) of public keys
    # that can be matched based on the kid (key id) and used to verify
    # payloads signed by the same Notary in other endpoints.
    # In addition to the fields required for a JSON Web Key (JWK),
    # a Base64URL encoded PEM is included for ease of use.
```
@kalilsn what do you think of this adjustment?